src/file/open.php

```diff
@@ -23,7 +23,7 @@ function open_write_only_nd(
   WriteMode $mode = WriteMode::OPEN_OR_CREATE,
 ): NonDisposableWriteHandle {
   return
-    NonDisposableWriteHandle::__createInstance_IMPLEMENTATION_DETAIL_DO_NOT_USE(
+    _Private\NonDisposableWriteHandle::__createInstance_IMPLEMENTATION_DETAIL_DO_NOT_USE(
```

Comment:
now they're in _Private, the constructor can be public instead; same change throughout